### ISSUE TITLE
Create a playlist file after downloading songs from a playlist URL

### ISSUE THREAD

**Issue by iamprem** [TheWicklowWolf/Syncify#3]
Currently syncify downloads all songs from a playlist/album URL into a folder. I have to manually create a playlist file m3u for each folder. It would be nice if syncify auto creates this file after download.

Music servers like Navidrome and Jellyfin can read this file and show the playlist in the player.

I'm happy to send PR if you think this can be added as a feature.

**Comment by TheWicklowWolf** [comment]
I developed [PlaylistDir](https://github.com/TheWicklowWolf/PlaylistDir) to automate the creation of playlists. It works by generating an .m3u file for all music files within a subfolder of a specified directory.

I decided not to include playlist functionality in Syncify because I wasn't certain it would be useful for everyone. 
Also, if files are downloaded to a intermediate location, the playlists would not be valid. 

I also kept them separate to accommodate playlists that aren't part of Syncify, which would otherwise be excluded. Keeping them separate allows you to generate a playlists for one-time downloads, such as when using [SpotTube](https://github.com/TheWicklowWolf/SpotTube).

Haven't said that I'm not completely against the idea, but have a look at PlaylistDir and see if that suits.
